### PR TITLE
added functionality to bind to the src of the script

### LINKED ISSUE #57: Unable to bind script tag's src attribute
### Description

When using `<marked-element>` in a **template** and binding the `src` attribute of the `<script>` tag, the `src`attribute is always found empty in the  `<marked-element>` ready callback. The attribute seems to be read a bit early.

### Expected outcome

The `<marked-element>` should be able to read the `src` attribute properly and make the request, so the markdown file could be downloaded and displayed.

### Actual outcome

Actually the `src` attribute is not bound (or bound too late), and no markdown is displayed.

### Live Demo

https://jsbin.com/gaqoqoxuji/1/edit?html,output

### Steps to reproduce

See live demo snippet.

### Browsers Affected

All

### Proposal

What about read the `src` attribute in the attached callback instead ?

### PR DESCRIPTION
**Pull request by e111077**
fixes #57.

### CODE REVIEW

File: marked-element.html [hunk 1]
```diff
@@ -305,10 +314,13 @@
         // Note: if we are using the file:// protocol, the status code will be 0
         // for all outcomes (successful or otherwise).
         if (status === 0 || (status >= 200 && status < 300)) {
+          this.sanitize = true;
           this.markdown = e.target.response;
         } else {
           this._handleError(e);
         }
+
+        this.fire('marked-loadend', e);
```
Comment (e111077) [inline marked-element.html:337]: necessary because data binding does not have a `this.xhr` so you can't attach a `loadend` listener.

**Comment by stramel** [comment]
Can you update the documentation so it notes that we handle the src attribute changing? Previously, all interactions with script[type="text/markdown"] was static.

**Comment by stramel** [comment]
So not sure if we want to continue maintaining essentially the same documentation in both the README and inline with the component or not. Looks like you missed the inline documentation.

Also, that documentation note applied to `<script type="text/markdown">...</script>` without the `src` attribute as well,  which I believe is static.

**Comment by e111077** [comment]
Oh I didn't catch that, and that's a good point. Reverting and documenting more-clearly.

**Review by stramel — APPROVED** [review]

**Pull request merged**